### ISSUE TITLE
Problems about bootstrapped CE loss

### ISSUE THREAD

**Issue by JulianJuaner** [PingoLH/FCHarDNet#45]
Hi, I reference your HarDNet structure to train cityscapes (on my own segmentation codebase, which could match the performance of almost all popular segmentation networks).
I use your pre-trained ImageNet network and follow almost all configs (except l use CE loss) as you set in this repo, but just got a 70.49 MIOU on the validation set (single scale inference).
But when I try to use the bootstrapped CE loss you implemented, I find that the loss is always in a very small range (around 0.15) and training pixel accuracy does not have a smooth increase. Is it normal when you train your model? 
I'll appreciate it if you answer this question!

**Comment by JulianJuaner** [comment]
Sorry I got that. It is because of the different normalization methods when the `weights` param passed in nn.CrossEntropy when the reduction is set to "none". I got a 75.25 mIoU now, thanks.
